### PR TITLE
Add adidas case study

### PR DESCRIPTION
**Pull request by jkld11**
This is the first case study using the new CSS that was just merged. Per discussion, you'll be able to see what the new styling looks like on the preview site. Thanks!

### CODE REVIEW

**Comment by jkld11** [comment]
I've signed the CLA. Thanks!

File: content/en/case-studies/adidas/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+​
+<h2>Impact</h2>
+    Just six months after the project began, 100% of the adidas e-commerce site was running on Kubernetes. Load time for the e-commerce site was reduced by half. Releases went from every 4-6 weeks to 3-4 times a day. With 4,000 pods, 200 nodes, and 80,000 builds per month, adidas is now running 40% of its most critical, impactful systems on its cloud native platform.
+</div>
+</div>
+</section>
+<div class="article image overlay" style="background-image: url('/images/Adidas2.png');padding-top:2% !important">
+  <div class="quotetext">
+    "For me, Kubernetes is a platform made by engineers for engineers. It’s relieving the development team from tasks that they don’t want to do, but at the same time giving the visibility of what is behind the curtain, so they can also control it."
+<p><span style="font-size:14px;letter-spacing:2px;text-transform:uppercase;">- FERNANDO CORNAGO, SENIOR DIRECTOR OF PLATFORM ENGINEERING AT ADIDAS</span></p>
```
Comment (sftim) [inline content/en/case-studies/adidas/index.html]: If this is text-transformed to upper case, what case should the text be in source code?

**Comment by alexcontini** [comment]
@sftim  sorry for all the messages! I created another PR to update the css so we can remove the inline styling, and then Julie updated this HTML file to remove the styling, but we are now getting failures about a png in the folder (Chris converted those to svgs so that shouldn't be happening) - do you know why this is happening?

Sorry once we figure out one I promise this will be much smoother!!

File: content/en/case-studies/adidas/index.html [hunk 2]
```diff
@@ -68,13 +68,13 @@ <h2>In recent years, the adidas team was happy with its software choices from a
 </section>
 ​
 ​
-  <div class="article image overlay" style="background-image: url('/images/Adidas3.png');padding-top:2% !important">
+  <div class="article image overlay" style="background-image: url('/images/Adidas3.png');">
   <div class="quotetext">
```
Comment (kbhawkey) [inline content/en/case-studies/adidas/index.html]: @jkld11 , is this url to .png file correct? Should there be a corresponding .svg file?

Comment (jkld11) [reply content/en/case-studies/adidas/index.html]: @kbhawkey Hi, yes, those urls are correct. They are the images that are the background for the banners.

Comment (kbhawkey) [reply content/en/case-studies/adidas/index.html]: The error message:
8:28:59 PM: ERROR 2019/11/12 01:28:59 Case Studies: Missing logo for adidas. Put a PNG with the word 'logo' in the filename into "case-studies/adidas/"

**Pull request closed without merging**